OpenSSL 3.6.1 broke our big endian configuration with a big asm refactor
in https://github.com/openssl/openssl/commit/8f24a148e6f112045cf110c3e3f4e2415d9c11fe.

Temporary workaround until the asm is fixed.

OpenSSL issue: https://github.com/openssl/openssl/issues/29845

--- a/include/crypto/aes_platform.h
+++ b/include/crypto/aes_platform.h
@@ -77,7 +77,11 @@ void AES_xts_decrypt(const unsigned char *inp, unsigned char *out, size_t len,
 #define HWAES_xts_decrypt aes_p8_xts_decrypt
 #endif /* OPENSSL_SYS_MACOSX */
 #if !defined(OPENSSL_SYS_AIX) && !defined(OPENSSL_SYS_MACOSX)
+#if defined(__BIG_ENDIAN__)
+#define PPC_AES_GCM_CAPABLE 0
+#else
 #define PPC_AES_GCM_CAPABLE (OPENSSL_ppccap_P & PPC_MADD300)
+#endif
 #define AES_GCM_ENC_BYTES 128
 #define AES_GCM_DEC_BYTES 128
 size_t ppc_aes_gcm_encrypt(const unsigned char *in, unsigned char *out,
